src/Control/Email/Email.php

```diff
@@ -316,430 +221,142 @@ private function getDefaultFrom(): string
-     * @return $this
-     */
-    public function addCC($address, $name = null)
-    {
-        $address = $this->sanitiseAddress($address);
-        $this->getSwiftMessage()->addCc($address, $name);
-
-        return $this;
-    }
-
-    /**
-     * @return array
-     */
-    public function getBCC()
-    {
-        return $this->getSwiftMessage()->getBcc();
-    }
-
-    /**
-     * @param string|array $address
-     * @param string|null $name
-     * @return $this
-     */
-    public function setBCC($address, $name = null)
-    {
-        $address = $this->sanitiseAddress($address);
-        $this->getSwiftMessage()->setBcc($address, $name);
-
-        return $this;
-    }
-
-    /**
-     * @param string|array $address
-     * @param string|null $name
-     * @return $this
-     */
-    public function addBCC($address, $name = null)
-    {
-        $address = $this->sanitiseAddress($address);
-        $this->getSwiftMessage()->addBcc($address, $name);
-
-        return $this;
-    }
-
-    /**
-     * @return mixed
-     */
-    public function getReplyTo()
-    {
-        return $this->getSwiftMessage()->getReplyTo();
-    }
-
-    /**
-     * @param string|array $address
-     * @param string|null $name
-     * @return $this
-     */
-    public function setReplyTo($address, $name = null)
-    {
-        $address = $this->sanitiseAddress($address);
-        $this->getSwiftMessage()->setReplyTo($address, $name);
-
-        return $this;
-    }
-
-    /**
-     * @param string|array $address
-     * @param string|null $name
-     * @return $this
-     */
-    public function addReplyTo($address, $name = null)
-    {
-        $address = $this->sanitiseAddress($address);
-        $this->getSwiftMessage()->addReplyTo($address, $name);
-
-        return $this;
-    }
-
-    /**
-     * @return string
-     */
-    public function getSubject()
-    {
-        return $this->getSwiftMessage()->getSubject();
-    }
-
-    /**
-     * @param string $subject The Subject line for the email
-     * @return $this
-     */
-    public function setSubject($subject)
-    {
-        $this->getSwiftMessage()->setSubject($subject);
-
-        return $this;
-    }
-
-    /**
-     * @return int
-     */
-    public function getPriority()
-    {
-        return $this->getSwiftMessage()->getPriority();
-    }
-
-    /**
-     * @param int $priority
-     * @return $this
-     */
-    public function setPriority($priority)
-    {
-        $this->getSwiftMessage()->setPriority($priority);
-
-        return $this;
-    }
-
-    /**
-     * @param string $path Path to file
-     * @param string $alias An override for the name of the file
-     * @param string $mime The mime type for the attachment
-     * @return $this
+     * Get template data
+     *
+     * Hardcoded data - these used to be public methods in v4 when Email extended ViewableData:
+     * IsEmail: used in template detect if rendering an email template rather than a page template
```

Comment:
```suggestion
     * IsEmail: used to detect if rendering an email template rather than a page template
```
Is this actually used by any templates?

Reply:
Depends on the project I suppose. It seems very weird to me to actually use this, though retaining for the sake of backwards compatibility.

Reply:
It's technical debt if it's not being used by anything we have - but it's so minor that I'm fine with it being left in.